.travis.yml

```diff
@@ -88,6 +89,7 @@ jobs:
             <<: *STANDARD_TEST_JOB
             stage: Test
             php: 7.2
+            env: SYMFONY_DEPRECATIONS_HELPER=disabled PHP_CS_FIXER_TEST_USE_LEGACY_TOKENIZER=1 SYMFONY_VERSION="5.0.x-dev" MINIMUM_STABILITY=dev
```

Comment:
Symfony 5.0 stable is available now, we can use it rather than the `-dev` version.

Reply:
Indeed... Fixed :)

Reply:
Maybe `^5.0` instead of `~5.0.0`? Also the line with `$MINIMUM_STABILITY` isn't necessary anymore.